lib/voyager_helpers/liberator.rb

```diff
@@ -192,6 +192,7 @@ def get_availability(bibs, full=false)
               else
                 item = get_info_for_item(holding_item_ids.first, c, false)
                 availability[bib_id][mfhd_id][:temp_loc] = item[:temp_location] unless item[:temp_location].nil?
+                availability[bib_id][mfhd_id][:course_reserves] = get_courses(holding_item_ids, c).map(&:to_h)
```

Comment:
(get_availability gets called once per bib to drive the results page availability)